### PR TITLE
Fix crossdeploy (Windows target)

### LINKED ISSUE #652: crossdeploy not working on Linux (Windows target)
crossdeploy not working on Debian for windows target. I had cross compile executable that built with mxe project, the sample project had qml for graphical user interface but when the cqtdeployer had run with command below, it only copied the binary file to the target not qml files and not dll libraries found on the target.
```
cqtdeployer -bin qml-sample.exe -qmlDir ui/qml -targetDir /tmp -qmake /usr/bin/qmake.mingw64
```

**To Reproduce**
1. Pull `hivaiot/ci-qt-windows-target:dynamic` docker image to build cross compile for windows
2. Install cqtdeployer using dpkg
3. Run the command like that explained above
4. See the result

**Desktop (please complete the following information):**
 - OS: Debian
 - Version Buster

### PR DESCRIPTION
**Pull request by EndrII**
Fix #652

### CODE REVIEW

**Comment by EndrII** [comment]
@khozaei i prepare builds with this fix, check please this fix.

**Comment by EndrII** [comment]
@khozaei check please new [builds](https://quasarapp.ddns.net:3031/QuasarApp/CQtDeployer/f7576405e6791d5432e9b83b7c0aae14f70221d4/)

File: Deploy/deploycore.h [hunk 1]
```diff
@@ -303,6 +303,12 @@ class DEPLOYSHARED_EXPORT DeployCore
      */
     static QFileInfo findItem(const QString &file);
 
+    /**
+     * @brief getCaseSensitivity This method return case sensitivity for a @a checkedFile. Usualy return Qt::CaseSensiativy exept windows binaryes files like a dll and exe.
```
Comment (IgorekLoschinin) [inline Deploy/deploycore.h:307]:
```suggestion
     * @brief getCaseSensitivity This method return case sensitivity for a @a checkedFile. Usually return Qt::CaseSensiativy exept windows binaryes files like a dll and exe.
```

**Review by IgorekLoschinin — APPROVED** [review]

**Comment by EndrII** [comment]
i merge this PR because problem with missing libpng library does not apply to this task

**Pull request merged**